### PR TITLE
Added report to GEO Datasets

### PR DESCRIPTION
**Pull request by rokgomiscek**
GEODatasets: added Report

### CODE REVIEW

File: orangecontrib/bio/widgets3/OWGEODatasets.py [hunk 1]
```diff
@@ -639,6 +639,27 @@ def samplesinst(ex):
...
+        self.report_items("Data", [("Samples", self.currentGds['sample_count']),
+                                   ("Features", self.currentGds['feature_count']),
+                                   ("Genes", self.currentGds['gene_count'])])
+        self.report_name("Sample annotations")
+        subsets = {}
+        for subset in self.currentGds['subsets']:
+            if subset['type'] not in subsets:
+                subsets[subset['type']] = [(subset['description'], len(subset['sample_id']))]
+            else:
+                subsets[subset['type']].append((subset['description'], len(subset['sample_id'])))
```
Comment (kernc) [inline orangecontrib/bio/widgets3/OWGEODatasets.py]: You can reduce above four lines into a single last one if you define:

``` py
subsets = defaultdict(list)
```

File: orangecontrib/bio/widgets3/OWGEODatasets.py [hunk 2]
```diff
@@ -639,6 +639,27 @@ def samplesinst(ex):
...
+        self.report_html += "<ul>"
+        for type in subsets:
+            self.report_html += "<b>"+type+":</b></br>"
```
Comment (kernc) [inline orangecontrib/bio/widgets3/OWGEODatasets.py]: Missing spaces around operators.

**Comment by rokgomiscek** [comment]
Thanks, fixed.

**Comment by ajdapretnar** [comment]
@rokgomiscek Sorry for not mentioning this before, but could you add organism to the report - under 'Title'? Thanks! :)

**Comment by rokgomiscek** [comment]
@ajdapretnar Added

**Pull request merged**